src/super_gradients/training/models/predictions.py

```diff
@@ -53,3 +53,56 @@ def _validate_input(self, bboxes: np.ndarray, confidence: np.ndarray, labels: np
 
     def __len__(self):
         return len(self.bboxes_xyxy)
+
+
+@dataclass
+class PoseEstimationPrediction(Prediction):
```

Comment:
this file as well

Reply:
Moved to super_gradients.training.utils.predict in 16c29cc93eb8827a32626b4a9f8433a0f5b818ba